doc/examples/anim12/anim12.sh

```diff
@@ -21,7 +21,7 @@ FILT=21
 # INC=5.625
 # FILT=10.5
 # 1. Create preflight script to build data files needed in the loop
-cat << EOF > pre.sh
+cat << 'EOF' > pre.sh
 gmt begin
```

Comment:
This one is also problematic, because the `pre.sh` script need the expanded `$SIZE`.